CHANGELOG.md

```diff
@@ -8,7 +8,7 @@
 
 ### Breaking Changes
 
-- [#689](https://github.com/okta/okta-auth-js/pull/689) New methods `start` and `stop` are added to control `OktaAuth` as a service. TokenManager options `autoRenew`, and  `autoRemove` will only take effect after calling `start`.
+- [#689](https://github.com/okta/okta-auth-js/pull/689) New methods `start` and `stop` are added to control `OktaAuth` as a service.
```

Comment:
Also mention `isPending` has been removed here, and add it to the migration guide.